easypost/services/address_service.py

```diff
@@ -0,0 +1,73 @@
+from typing import (
+    Any,
+    Dict,
+    List,
+    Optional,
+    Union,
+)
+
+from easypost.easypost_object import convert_to_easypost_object
+from easypost.models.address import Address
+from easypost.requestor import (
+    RequestMethod,
+    Requestor,
+)
+from easypost.services.base_service import BaseService
+
+
+class AddressService(BaseService):
+    def __init__(self, client):
+        self.model_class = "address"
+
+    def create(
+        self,
+        verify: Optional[Union[Dict[str, Any], str, bool]] = None,
+        verify_strict: Optional[Union[Dict[str, Any], str, bool]] = None,
+        **params,
+    ) -> Address:
+        """Create an Address."""
+        url = self.class_url(self.model_class)
+
+        wrapped_params = {self.snakecase_name(self.model_class): params}  # type: Dict[str, Any]
+        if verify:
+            wrapped_params["verify"] = verify
+        if verify_strict:
+            wrapped_params["verify_strict"] = verify_strict
+
+        response, api_key = Requestor().request(method=RequestMethod.POST, url=url, params=wrapped_params)
+
+        return convert_to_easypost_object(response=response, api_key=api_key)
```

Comment:
Question: Can we remove the `api_key` as part of this PR?

Reply:
No, all the old functions still need it. I'll be cleaning that all up once I can delete the old logic